main.css

```diff
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500&family=Poppins:wght@500&display=swap');
+
+* {
+  padding: 0;
+  margin: 0;
+}
+
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  width: 95%;
+  height: 30px;
+  padding: 2%;
+}
```

Comment:
- [ ] Kindly to match the previous design, you can make the `width` 100% instead of 95% and make the `height` 50px instead of 30px.